wiki/Play_style/en.md

```diff
@@ -145,11 +145,11 @@ To dash, many players use one of the face buttons on their gamepad. Though, some
 
 ### Keyboard
 
-[External guide for Mechanical Keyboard](http://www.daskeyboard.com/blog/mechanical-keyboard-guide/)
+*Note: For more information about mechanical keyboards, see this [external guide for mechanical keyboards](http://www.daskeyboard.com/blog/mechanical-keyboard-guide/).*
 
 Keyboard was once the *only* way to play osu!mania until other play methods was supported over time.
 
-To play with a keyboard, players would press down the keys respective to their placement on the in-game keyboard. By default these keys are DFJK. The advantage of this play-style is that it can accommodate basically all the Keys setting.
+To play with a keyboard, players would press down the keys respective to their placement on the in-game keyboard. By default these keys are `D` `F` `J` `K`. The advantage of this play-style is that it can accommodate to basically all the keys settings.
```

Comment:
```suggestion
To play with a keyboard, players would press down the keys respective to their placement on the in-game keyboard. By default these keys are `D`, `F`, `J`, and `K`. The advantage of this play style is that it can accommodate to basically all the keys settings.
```

Reply:
There is a similar listing of keys in line 94

> *Note: For key-binding, the default is `Z` `X` `C` `V` (KDDK, or BRRB).*

Should this also be changed like this here?

Reply:
yeah i suppose